gradle/README.md

```
@@ -0,0 +1,269 @@
+line or in `gradle.properties`). The DSL takes precedence when both are set.
+
+```
+# gradle.properties
+systemProp.nisse.source.jgit.dynamicVersion=true
+systemProp.nisse.source.jgit.appendSnapshot=false
+```
+
+Or on the command line:
+
+```
+$ gradle build -Dnisse.source.jgit.dynamicVersion=true
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)